models/migrations/v71.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2018 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"github.com/go-xorm/xorm"
+)
+
+func addMustChangePassword(x *xorm.Engine) error {
+	type User struct {
+		ID                 int64 `xorm:"pk autoincr"`
+		MustChangePassword bool  `xorm:"NOT NULL DEFAULT 0"`
```

Comment:
Should be `NOT NULL DEFAULT false`

Reply:
Not sure why this review wasn't collapsed but it has been fixed in https://github.com/go-gitea/gitea/pull/4489/files#diff-c6d02501c53821722868adbdd6d59a34R14